packages/volto/src/helpers/ScrollToTop/ScrollToTop.jsx

```diff
@@ -22,12 +22,21 @@ class ScrollToTop extends React.Component {
     children: PropTypes.node.isRequired,
   };
 
+  state = {
+    isFirstClientUpdatePending: true,
+  };
   /**
    * @param {*} prevProps Previous Props
    * @returns {null} Null
    * @memberof ScrollToTop
    */
   componentDidUpdate(prevProps) {
+    // Skip the first client-side update that happens right after hydration
+    if (this.state.isFirstClientUpdatePending) {
+      this.setState({ isFirstClientUpdatePending: false });
+      return;
+    }
+
```

Comment:
@Faakhir30 We can do this a bit more simply without using component state. Instead, add a check for `&& prevProps.location !== undefined` after line 41. Could you make this change please?